db/re/item_db.txt

```diff
@@ -1020,12 +1020,12 @@
 1668,Sword_Stick,Sword Stick,5,10,,500,120:150,,,2,0x800200,63,2,2,4,80,1,10,{ bonus bAspdRate,10; },{},{}
 1669,Thanos_Staff,Thanatos Staff,5,10,,1000,100:200,,1,1,0x00018314,56,2,2,4,120,1,10,{ bonus bInt,6; bonus bVit,6; bonus bLuk,-6; bonus bHealPower,15; bonus bMagicHPGainValue,500; bonus bMagicSPGainValue,50; bonus2 bHPLossRate,100,10000; },{},{ heal -1000,0; }
 1670,RWC_Memory_Staff,RWC Memory Staff,5,20,,500,25:30,,1,1,0x00818315,63,2,2,3,1,1,10,{ .@r = getrefine(); bonus bMatk,30*(.@r/3); if(.@r>=6) bonus2 bMagicAddClass,Class_All,(.@r>=9?10:5); if(.@r>=9) bonus4 bAutoSpell,"HW_MAGICPOWER",1,10,0; },{},{}
-1671,Staff_Of_Evil_Slayer,Devil Won Staff,5,10,,800,30:155,,,1,0x00818315,63,2,2,3,100,1,10,{ bonus2 bAddRace,RC_Undead,10; bonus2 bAddRace,RC_Demon,10; bonus bMatkRate,10; },{},{}
+1671,Staff_Of_Evil_Slayer,Devil Won Staff,5,10,,800,30:155,,,1,0x00818315,63,2,2,3,100,1,10,{ bonus2 bAddRace,RC_Undead,10; bonus2 bAddRace,RC_Demon,10; .@r = getrefine(); if(.@r>=9) bonus bMatkRate,5; if(.@r>=12) bonus bMatkRate,12; },{},{}
```

Comment:
I think @Lemongrass3110 commented on this in your previous PR. The second `bMatkRate` bonus should be 7, not 12.

Reply:
Before was 7 and he asked for change.

Reply:
No, he was saying they shouldn't stack. Currently how you have it setup if you have 12 or more refine you will get Matk + 5 and Matk + 12 (which 12 isn't right). You should get one (5) or the other (7).

So it should be like: `.@r = getrefine(); if(.@r>=12) bonus bMatkRate,7; else if(.@r>=9) bonus bMatkRate,5;`

Reply:
Thank you @aleos89 
I did it, but now I need to know how I can "send" the changes...
I'm using GitHub Desktop and Atom to edit, but I don't know how I can proceed. :/

![git](https://user-images.githubusercontent.com/34192842/35745804-ab43da00-082b-11e8-8a56-704bdafd3a59.png)

![git2](https://user-images.githubusercontent.com/34192842/35745813-b28a9254-082b-11e8-9423-b53016f68742.png)

Reply:
On the bottom left I'm assuming (I've never used Github Desktop). Give it a summary (title) and a description and click the "Commit to pr/2828" button.

Reply:
Thank you very much @aleos89 
I did it, but I don't know if works!
It's saying here that I commited seven minutes ago.
Is just it or I need do something more?

![git](https://user-images.githubusercontent.com/34192842/35749923-4e0300c8-083a-11e8-933f-5700d13d8065.png)

I prefer the Desktop Version with Atom, it seems to be more easy to edit and use.

Could you see the PM that I send to you in rA forums, about others changes...
Sorry for any mistake. I'm trying to learn about Github, I've never used this.

Reply:
you need ti publish it 
![image](https://user-images.githubusercontent.com/5981261/35750136-acc7610e-085c-11e8-8089-6cd4d62f8937.png)

Reply:
I can't @sader1992. :-(

![git2](https://user-images.githubusercontent.com/34192842/35750589-a50c9274-083c-11e8-9e80-c4b5563d47fe.png)